.github/workflows/new-cfe-notifications.yml

```diff
@@ -31,33 +30,57 @@ jobs:
     if: github.event.label.name == 'CFE Approved'
     runs-on: ubuntu-20.04
     steps:
-      - name: Extract PR URL from issue body
+      - name: Extract PR number from issue body
         id: extract-pr
+        env:
+          BODY: ${{ github.event.issue.body }}
         run: |
-          PR_URL=$(echo "${{ github.event.issue.body }}" | grep -oP 'https://github.com/[^/]+/[^/]+/pull/\d+')
-
-          if [[ -z "$PR_URL" ]]; then
-            echo "No valid PR URL found in the issue body. Exiting."
+          # Use Bash regex to extract the PR number from the pull request URL
+          if [[ "$BODY" =~ https:\/\/github\.com\/[a-zA-Z0-9_\-]+\/[a-zA-Z0-9_\-]+\/pull\/([0-9]+) ]]; then
```

Comment:
So it would only take the first PR URL, as if someone follows the template, it should pick the correct URL and release numbers. That said, I have changed my approach and use the JS script now.